src/runner_manager.py

```diff
@@ -90,10 +89,15 @@ def __init__(
         self.session.mount("http://", adapter)
         self.session.mount("https://", adapter)
         if self.proxies:
+            proxy_dict = {
+                "http_proxy": self.proxies.http,
+                "https_proxy": self.proxies.https,
+                "no_proxy": self.proxies.no_proxy,
+            }
             # setup proxy for requests
-            self.session.proxies.update(self.proxies)
+            self.session.proxies.update(proxy_dict)
```

Comment:
`requests` proxy configuration is not keyed with `http_proxy` or `https_proxy`, it's keyed with `http` and `https`. And I don't remember there's a `no_proxy` for the proxy configuration.

See: https://requests.readthedocs.io/en/latest/user/advanced/#proxies
```python
import requests

proxies = {
  'http': 'http://10.10.1.10:3128',
  'https': 'http://10.10.1.10:1080',
}

requests.get('http://example.org', proxies=proxies)
```

Reply:
Thanks, good catch, seems to be wrongly refactored by me.

Reply:
requests seem to support `no_proxy` since 2.14.0 https://pypi.org/project/requests/2.17.2/

Reply:
After looking at the code again, I think setting the proxy vars here and below is no longer necessary, because we have lines 76-81 which set the env vars for the process and its childs, and requests and urllib seem to respect this. (Otherwise the integration tests should have failed too, because I used the wrong vars here).

@yhaliaw wdyt?